spec/services/reporters/event_service_reporter_spec.rb

```diff
@@ -5,17 +5,12 @@
 RSpec.describe Reporters::EventServiceReporter do
   let(:subject) { described_class.new }
 
-  let(:client) { instance_double(Dor::Services::Client::Events) }
```

Comment:
I don't understand it either. It was failing for me, thats why I removed it.

Reply:
something something dependency caching?  🤷  perplexing.

but then i also still don't get why the highlighted expectation works, since `client` is no longer defined anywhere in that file, as far as i can tell?

Reply:
the underlying issue and the PRs that fixed it, for future code archaeologists:
* https://github.com/sul-dlss/preservation_catalog/issues/1914
* https://github.com/sul-dlss/preservation_catalog/pull/1915
* https://github.com/CircleCI-Public/ruby-orb/issues/86